kolibri/core/assets/test/views/CoreBase/ScrollingHeader.spec.js

```diff
@@ -19,126 +19,4 @@ describe('Scrolling header movement logic', () => {
     const wrapper = createWrapper();
     expect(wrapper.exists()).toBe(true);
   });
-  it('should be pinned to the top by default', () => {
```

Comment:
Should we write a new suite of tests for this feature? Is there a clear way to accomplish automated testing for it?

Reply:
Yeah, I can add some basic tests for this. We can't really simulate actual scrolling, but do something to the old tests where pass in different props and make sure the component is doing the right thing in each case.